dev/managedservicesplatform/internal/resource/alertpolicy/alertpolicy.go

```diff
@@ -105,12 +105,17 @@ type ResponseCodeMetric struct {
 	Duration     *string
 }
 
-type SeverityLevel string
+// MQL for alerting when a MQL query exceeds a threshold
+type MQL struct {
+	Query    string
+	Duration *string
+}
 
-const (
-	SeverityLevelWarning  = "WARNING"
-	SeverityLevelCritical = "CRITICAL"
-)
+// PromQL for alerting when a PromQL query exceeds a threshold
```

Comment:
```suggestion
// PromQL for alerting when a PromQL query exceeds a threshold.
```

And same question to the above.